.travis.yml

```diff
@@ -6,7 +6,8 @@ rust:
   - nightly
 matrix:
   allow_failures:
-    - rust: nightly
+    - rust: stable
+    - rust: beta
```

Comment:
But they'll stop failing as soon as the features we need are stabilized. Should we still remove them?